wagtail/wagtailsnippets/tests.py

```diff
@@ -131,6 +131,17 @@ def test_simple(self):
         response = self.get()
         self.assertEqual(response.status_code, 200)
         self.assertTemplateUsed(response, 'wagtailsnippets/snippets/create.html')
+        self.assertNotContains(response, '<a href="#advert" class="active">Advert</a>')
+        self.assertNotContains(response, '<a href="#other" class="">Other</a>')
```

Comment:
Could you also assert that there is no `<ul class="tab-nav merged">` in the output?